train_hierarchialXC.py

```diff
@@ -23,8 +28,14 @@
 # training configuration arguments
 parser.add_argument('--seed', type=int, default=None,
                     help='Manually set the seed for the experiments for reproducibility')
-parser.add_argument('--plot', action='store_true',
-                    help='Option to plot the loss variation over iterations')
+parser.add_argument('--interval', type=int, default=-1,
+                    help='Interval between two status updates on training')
+parser.add_argument('--input_ae_dim', type=int, default=500,
```

Comment:
Set the default to -1. If -1, then no dimensionality reduction is done on the inputs.

Reply:
Setting this to -1, is leading OOM killed errors.

Reply:
No, I am saying is let the auto-encoding be done based on requirements. If auto-encoding is required, we give a positive val, else we leave it.

Reply:
Mention the default value in the description.